payjoin/src/send/error.rs

```diff
@@ -103,7 +105,9 @@ impl fmt::Display for ValidationError {
             FeeContributionPaysOutputSizeIncrease => write!(f, "fee contribution pays for additional outputs"),
             FeeRateBelowMinimum =>  write!(f, "the fee rate of proposed transaction is below minimum"),
             #[cfg(feature = "v2")]
-            V2(e) => write!(f, "v2 error: {}", e),
+            HpkeError(e) => write!(f, "v2 error: {}", e),
```

Comment:
I'm not the happiest with this error string, but we should be reconsidering the context we want to keep with send::Error anyhow in a later issue.